### PR TITLE
Instructions for using Caddy as proxy server with Minio.

### PR DESCRIPTION
**Pull request by koolhead17**
Additions:
1. Created directory minio
2. Added ``README.md`` and  ``Caddyfile``

### CODE REVIEW

File: minio/Caddyfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+your.public.com {
+    proxy / localhost:9000 {
+        proxy_header Host {host}
+        proxy_header X-Real-IP {remote}
+        proxy_header X-Forwarded-Proto {scheme}
```
Comment (mholt) [inline minio/Caddyfile]: FYI, as of 0.9 beta these 3 lines can be replaced with one preset, "transparent".

```plain
proxy / localhost:9000 {
    transparent
}
```

File: minio/Caddyfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    }
+}
+
```
Comment (mholt) [inline minio/Caddyfile]: This Caddyfile has just one site in it so it doesn't require nesting. Could be as simple as:

```plain
your.example.com

proxy / localhost:9000 {
    transparent
}
```

File: minio/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Minio Object storage with Caddy server
+
+[Minio](https://www.minio.io) is an Open Source object storage. If you want to inherent webserver features like load balancing, IP filtering and monitoring you can use Caddy in front of Minio.
```
Comment (mholt) [inline minio/README.md]: "inherit"

**Comment by koolhead17** [comment]
@mholt updated the PR with your suggestions.

**Comment by mholt** [comment]
Thanks!

**Pull request merged**